spec/models/factory_log_spec.rb

```diff
@@ -12,13 +12,13 @@
       FactoryLog.logging(inspected_factory2, 2.1)
       expect(FactoryLog.ranking).to eq [
                                          {
-                                            factory: factory1,
+                                            factory: factory1.to_s,
                                             total_time: 1.0 + 1.1 + 1.0,
                                             count: 3,
                                             average_time: (1.0 + 1.1 + 1.0) / 3
                                          },
                                          {
-                                            factory: factory2,
+                                            factory: factory2.to_s,
```

Comment:
Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.